### ISSUE TITLE
Reopen issue #2190

### ISSUE THREAD

**Issue by fluffy-unicorn-71** [sweetalert2/sweetalert2#2198]
Just writing that you don't understand what I say and then closing the ticket seems a bit rude. You have done it twice. Ask for a clarification if you need but don't just close the ticket. I'm opening it to help YOU making YOUR product better, please respect that.

Let me explain again: when you click a button that triggers a popup, the popup is being rendered/loaded. If the popup e.g. contains data that is being fetched from a database, this rendering may take some time. During that time, there is a loader displayed. The default loader is a spinning blue circle. According to your documentation, if I want to customize that loader I can do so with the parameters customClass.loader and loaderHtml. **The problem is that it does not work!**

In your first reply, you sent me a link to a recipe for customizing the loader. But as I wrote to you, that customization only works for the loader that is displayed when the confirm button in the popup is clicked. It does NOT work for the loader that is displayed when the popup is rendered.

Please see the codepen snippet I sent you and ask for more clarification if you need.

**Comment by limonte** [comment]
You sound like I owe you something :) I still do not understand what you're talking about. Show me the code which demonstrates the issue that you're describing.

**Comment by fluffy-unicorn-71** [comment]
The code you ask for was attached as a link to a codepen project in my first bug report. 

That code DOES reproduce the error, but since the content of the popup loads almost instantly, the loader may not be visible at all. If you instead had some other content in the popup that took longer time to fetch/render, the loader would be displayed and you would be able to clearly see the problem.

Please see the attached video. The content of that popup is dynamically fetched from a database with a slower connection. The blue loader that is visible in the video is the one where the customization has no effect. However, that same loader is also displayed when the confirm button is clicked and in that case, customizations works just as described in the docs.

**In other words: the blue loader is displayed in the following cases:
a) popup loading (as seen in the video)
b) when clicking the confirm button. 
However, the customization as described in your docs only have effect on b), not a).**

Please let me know if you need more clarification.

https://user-images.githubusercontent.com/34483851/111693105-7ce32200-8830-11eb-8f09-f476d89d741a.mov

**Comment by limonte** [comment]
When I'm asking for the code, you show the code. Not words, not videos. You're blocked now. Good luck.

**Comment by indrif** [comment]
@limonte what's up with the attitude? Can we please get a solution to the `loaderHtml`-attribute.

I have a problem with this as well. Tried changing loaderHtml in many ways but unable to get an effect in any way.

![image](https://user-images.githubusercontent.com/6198331/111920391-e82a2f80-8a8e-11eb-9df7-0e407e91363f.png)

Also there are no examples using this attribute on your page.

But, please be respectful to each other here at Github. Especially your community @limonte. No need to be rude, we're just asking for help for an awesome library.

**Comment by limonte** [comment]
@indrif please create a new issue **AND PLEASE FOLLOW THE ISSUE TEMPLATE**
